apis/core/v1beta1/openstackcontrolplane_types.go

```diff
@@ -82,6 +83,10 @@ type OpenStackControlPlaneSpec struct {
 	// +kubebuilder:validation:Optional
 	// Nova - Parameters related to the Nova services
 	Nova NovaSection `json:"nova,omitempty"`
+
+	// +kubebuilder:validation:Optional
+	// ExtraMounts containing conf files and credentials
```

Comment:
Please document that his will be ignored if the per service level extraMounts is defined.

Reply:
Done